src/components/atoms/Form/Form.component.tsx

```diff
@@ -20,14 +18,12 @@ export interface FormInterface
     AntFormType {}
 
 const FormComponent = React.forwardRef<HTMLDivElement, FormProps>((props, ref) => {
-  const { children, formRef, style, formStyle, responsiveVisibility, isPreview, ...rest } = props
+  const { children, responsiveVisibility, isPreview, ...rest } = props
   return (
     <VisibilityComponent visibility={responsiveVisibility} isPreview={isPreview}>
-      <div ref={ref} style={{ ...style, height: '100%' }}>
-        <AntForm {...rest} style={formStyle} ref={formRef}>
-          {children}
-        </AntForm>
-      </div>
+      <AntForm {...rest} ref={ref as any}>
+        {children}
+      </AntForm>
```

Comment:
@zrg-team Is this form composable via Jitera Editor? If so, removing the wrapper will raise error since the value of `ref` is not instanc eof HTMLElement

Reply:
@jefrydco i changed it to box